src/main/java/com/dynatrace/loadrunner/converter/CBodyFilePatcher.java

```diff
@@ -0,0 +1,35 @@
+package com.dynatrace.loadrunner.converter;
+
+import java.util.Set;
+
+import com.dynatrace.loadrunner.config.Mode;
+import com.google.common.collect.Sets;
+
+final class CBodyFilePatcher extends AbstractBodyFilePatcher {
+
+	private static final char C_PARAM = '"';
+	private static final String C_REGEX = "\\s*\\(\\s*([\"']).*\\1\\s*\\);";
+	private static final String C_TRANSACTION_START = "lr_start_transaction";
+	private static final String C_TRANSACTION_END = "lr_end_transaction";
+
+	private final static Set<String> C_REQUEST_KEYWORDS = Sets.newHashSet("web_url", "web_link", "web_image",
```

Comment:
just as a reminder - it is more compliant to write `static final` than `final static` ;)